### PR TITLE
Added docs page for the EnumToIntConverter...

### PR DESCRIPTION
**Pull request by CliffAgius**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] enum-To-int Converter #59 - https://github.com/CommunityToolkit/Maui/issues/59

### CODE REVIEW

**Comment by PRMerger7** [comment]
@CliffAgius : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by CliffAgius** [comment]
looks like you beat me to it with the file extention missing. Thanks buddy.

**Comment by PRMerger-2** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by bijington** [comment]
> looks like you beat me to it with the file extention missing. Thanks buddy.

No worries, it seemed easier to fix than point out the issue 😄

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/enumtointconverter.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        Content = new StackLayout
+    {
+      Children = 
+      {
+        new Picker()
+          .Bind(Picker.ItemSource, nameof(ViewModel.AllStates)
+          .Bind(Picker.SelectedIndex, nameof(ViewModel.SelectedState),
+
+          new Label()
+            .Bind(Label.TextPropoerty, nameof(ViewModel.SelectedState), converter: new EnumToIntConverter()),
```
Comment (bijington) [inline docs/maui/converters/enumtointconverter.md:69]: This looks remarkably similar to the `C# Markup` section below 😄. 

I am happy to try and convert to the standard way of doing things if it helps

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by bijington — COMMENTED** [review]
@CliffAgius thanks for this! There is only the one comment but I am happy to help fix that if required.

I hope you don't mind but I went ahead and ordered the `TOC.yml` and `index.md` contents alphabetically

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/enumtointconverter.md (hunk 1 above)
Comment (CliffAgius) [reply docs/maui/converters/enumtointconverter.md:69]: I cut and paste that section from the Proposal as I don't tend to build my UI in C#, I must admit I am a little confused as to the difference between the C# and Markup so happy to be shown what the difference is.

Comment (CliffAgius) [reply docs/maui/converters/enumtointconverter.md:69]: Actually I see what you mean by the difference and think I understand what your looking for. Let me have another crack at it in the morning.

Comment (bijington) [reply docs/maui/converters/enumtointconverter.md:69]: OK thanks, let me know how you get on. Basically the thought behind the C# section and then the C# Markup section is to show how great the .Markup package is for developers that like to build their UI in C# only. Actually I suspect this example will show just how much simpler the C# Markup package makes it.

File: docs/maui/converters/enumtointconverter.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+using CommunityToolkit.Maui.Markup;
+
+class EnumToIntConverterPage : ContentPage
+{
+    public EnumToIntConverterPage()
+    {
+        Content = new StackLayout {
+          new Picker()
+            .Bind(Picker.ItemSource, nameof(ViewModel.AllStates)
+            .Bind(Picker.SelectedIndex, nameof(ViewModel.SelectedState),
```
Comment (jfversluis) [inline docs/maui/converters/enumtointconverter.md:89]: Should this be ItemsSourceProperty and SelectedIndexProperty?

File: docs/maui/converters/enumtointconverter.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+          new Label()
+            .Bind(Label.TextPropoerty, nameof(ViewModel.SelectedState), converter: new EnumToIntConverter()),
```
Comment (jfversluis) [inline docs/maui/converters/enumtointconverter.md:92]: Typo in TextProperty

```suggestion
            .Bind(Label.TextProperty, nameof(ViewModel.SelectedState), converter: new EnumToIntConverter()),
```

File: docs/maui/converters/enumtointconverter.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+{
+    public EnumToIntConverterPage()
+    {
+        Content = new StackLayout
+    {
+      Children = 
+      {
+        new Picker()
+          .Bind(Picker.ItemSource, nameof(ViewModel.AllStates)
+          .Bind(Picker.SelectedIndex, nameof(ViewModel.SelectedState),
```
Comment (jfversluis) [inline docs/maui/converters/enumtointconverter.md:66]: Should this be ItemsSourceProperty and SelectedIndexProperty?

File: docs/maui/converters/enumtointconverter.md (hunk 1 above)
Comment (jfversluis) [inline docs/maui/converters/enumtointconverter.md:69]: Typo in TextProperty

```suggestion
            .Bind(Label.TextProperty, nameof(ViewModel.SelectedState), converter: new EnumToIntConverter()),
```

**Review by jfversluis — CHANGES_REQUESTED** [review]

**Pull request closed without merging**